goal_src/jak3/engine/ai/enemy.gc

```diff
@@ -777,8 +777,10 @@
   (none)
   )
 
+;; ERROR: Function may read a register that is not set: t2
 (defmethod set-ground-pat! ((this enemy) (arg0 collide-query) (arg1 collide-spec) (arg2 float) (arg3 float) (arg4 float))
-  (when (find-ground (-> this root) arg0 arg1 arg2 arg3 arg4)
+  (local-vars (t2-0 none))
+  (when (find-ground (-> this root) arg0 arg1 arg2 arg3 arg4 (the-as process t2-0))
```

Comment:
hm, this use of t2 seems like a bug... maybe this function needs another argument?